### PR TITLE
SentenceTransformer feature added

### PR DESCRIPTION
**Pull request by payonear**
SentenceTransformer added for alternative semantic similarity search. In Telegram bot interface named as USE.

### CODE REVIEW

File: support_model.py [hunk 1]
```diff
@@ -8,13 +8,14 @@
...
 else:
-    from ml_models import elastic_search_baseline, bert_emb_baseline, bpe_baseline
+    from ml_models import elastic_search_baseline, bert_emb_baseline, bpe_baseline, sentence_transformer_baseline
 
 
 class ModelNames(enum.Enum):
     ELASTIC = 'elastic'
     BERT = 'bert'
     BPE = 'bpe'
+    ST = 'use'
```
Comment (submaps) [inline support_model.py]: May be `ModelNames.USE` instead of `ModelNames.ST`?

File: ml_models/sentence_transformer_baseline.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        names_sparse_matrix = []
+        for i in tqdm(range(len(data))):
+            try:
+                names_sparse_matrix.append(self.sentence_embedding(data[i]))
+            except:
+                names_sparse_matrix.append(np.zeros(self.FEATURE_SIZE))
+        return names_sparse_matrix
+
+
+class STIndexer:
```
Comment (submaps) [inline ml_models/sentence_transformer_baseline.py:57]: Is it possible to make ʻAbstract Class` or a general class for this model and the previous one (BertIndexer)?

File: ml_models/sentence_transformer_baseline.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
+from functools import wraps
+import os
+import pandas as pd
+import nmslib
+import numpy as np
+import torch
+from tqdm import tqdm
+from sentence_transformers import SentenceTransformer
```
Comment (submaps) [inline ml_models/sentence_transformer_baseline.py:7]: Please add this dependency to requirements.txt

File: ml_models/sentence_transformer_baseline.py [hunk 4]
```diff
@@ -0,0 +1,156 @@
+from functools import wraps
+import os
+import pandas as pd
+import nmslib
+import numpy as np
+import torch
```
Comment (submaps) [inline ml_models/sentence_transformer_baseline.py]: unused import

File: support_model.py [hunk 5]
```diff
@@ -8,13 +8,14 @@
 if test_mode:
     elastic_search_baseline, bert_emb_baseline, bpe_baseline = 1, 2, 3
```
Comment (submaps) [inline support_model.py]: You should add import `sentence_transformer_baseline` for test_mode

File: ml_models/sentence_transformer_baseline.py (hunk 3 above)
Comment (payonear) [reply ml_models/sentence_transformer_baseline.py:7]: added in the following file

File: ml_models/sentence_transformer_baseline.py [hunk 6]
```diff
@@ -3,7 +3,6 @@
 import pandas as pd
 import nmslib
 import numpy as np
-import torch
```
Comment (payonear) [inline ml_models/sentence_transformer_baseline.py]: unused import fix

File: support_model.py [hunk 7]
```diff
@@ -6,7 +6,7 @@
 
 test_mode = not os.environ.get('TOKEN')
 if test_mode:
-    elastic_search_baseline, bert_emb_baseline, bpe_baseline = 1, 2, 3
+    elastic_search_baseline, bert_emb_baseline, bpe_baseline, sentence_transformer_baseline = 1, 2, 3, 4
```
Comment (payonear) [inline support_model.py:9]: Sentence Transformer imported for test mode

File: support_model.py [hunk 8]
```diff
@@ -15,7 +15,7 @@ class ModelNames(enum.Enum):
     ELASTIC = 'elastic'
     BERT = 'bert'
     BPE = 'bpe'
-    ST = 'use'
+    USE = 'use'
```
Comment (payonear) [inline support_model.py:18]: As suggested, ST renamed to USE for clarity.

File: support_model.py [hunk 9]
```diff
@@ -48,7 +48,7 @@ def get_answer(query, use_lower=True, use_keywords=False, use_remove_stopwords=F
             answer_list = bert_emb_baseline.get_answer(query)
         if model_name == ModelNames.BPE:
             answer_list = bpe_baseline.get_answer(query)
-        if model_name == ModelNames.ST:
+        if model_name == ModelNames.USE:
```
Comment (payonear) [inline support_model.py:51]: As suggested, ST renamed to USE for clarity.

File: ml_models/sentence_transformer_baseline.py (hunk 2 above)
Comment (submaps) [reply ml_models/sentence_transformer_baseline.py:57]: We have decided to resolve it problem in next tasks

**Pull request merged**